SADL/AM-CDM-part.sadl

```diff
@@ -0,0 +1,27 @@
+uri "http://kdl.ge.com/AM-CDM-part" alias AM-CDM-part version "$Revision:$ Last modified on   $Date:$".
+
+Part (note "Instances of part designs in the as-built state.") is a MaterialStock,
```

Comment:
It is arguable if Part should be considered as "material".

Reply:
Do you have an alternative suggestion for this?

Reply:
Parts are made of a set of materials. Considering the additive focus of this model, the idea of "parts are material stock" is outlandish, but not wrong. I think this may just be a naming issue, but what are we trying to represent here? do we have a use case in mind?

Reply:
Generally, when I have a problem like this where I am trying to model for group of entities (in this case, AM processes), I just look more generally. That said, I think there are two good options here:

### Option 1

Instead of looking at individual AM inputs, Feedstock, powder, etc., we need to clarify, "What is the input material of an AM process called?" I'd suggest "rawMaterial," "printingMaterial," "buildMaterial," or "inputMaterial." They doesn't sound very scientific/engineering-ish, but the AM-CDM is attempting to standardize over AM processes. This leads us to Option 2.

### Option 2

If we want to further specify terminology for AM processes, then we should do so through inheritance. Selecting one of the names above, such as printingMaterial, keep the definition of the class minimal, and derive additional subclasses with more specific naming and terminology.

printingMaterial (base class containing all AM input material)
- Powder (derived class containing powders)
- feedstock (derived class containing powders)
- wire (derived class containing powders)
- resin (derived class containing powders)
- filament (derived class containing powders)

Reply:
Should be assessed against pull request #17